### DISCUSSION TITLE
F1 hide names ?

### DISCUSSION THREAD

**Discussion by Iki1209** [community/community#120802] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, how to config for make invisible name/nametag of players when you do F1 please ? Cause when you make F1 to have no HUD and to take some screenshots, with TAB plugin nickanmes are shown and not hidden with F1 keybind so we can't do some screenshots without seeing the name of the another players on the screenshots, ty!

(I don't found another similar discussion but I'm new to github)

**Comment by manishpatel0** [comment]
1. Check the TAB plugin settings:  Many TAB plugins have options to control the visibility of player names or nicknames. Look for settings related to "nametags," "player labels," or "HUD elements" within the plugin's configuration menu.

2. Use external tools for screenshots: Some third-party screenshot tools allow editing the captured image after taking the screenshot. You could use such a tool to manually crop out or blur the player names before saving the final image

**Comment by Mahdi-Shabani** [comment]
First, press the "Print Screen" button, your keyboard may be displayed as "Prtscn". By pressing this key, all the things you see on your screen will be transferred to your clipboard in the form of a photo. If you have multiple monitors and want to capture only the current monitor you are working on, press the combination keys "Alt + PrtScn" at the same time.

In the next step, open Microsoft Paint. For this purpose, search for "Paint" in the start menu and then open the first result that appears.

When Paint is opened, hold the combination keys "Ctrl + V" at the same time so that the screenshot prepared in Paint appears. If you don't want to edit this screenshot, just hold the combination keys "Ctrl + S" simultaneously to save the screenshot.

(I hope I answered correctly)

**Comment by ebndev** [comment]
Hi @Iki1209,

Thanks for being a part of the GitHub Community, we're glad you're here!

This category is for conversations related to your GitHub profile page so I am going to close this discussion. I'm glad the users above were able to support you however for future discussions, you might want to try asking for help somewhere that focuses on Minecraft. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
